CMakeLists.txt

```diff
@@ -187,8 +187,17 @@ MESSAGE(STATUS "Cabana Revision = '${Cabana_GIT_COMMIT_HASH}'")
 # enable unit tests
 option(Cabana_ENABLE_TESTING "Build tests" OFF)
 if(Cabana_ENABLE_TESTING)
+  find_package(GTest 1.10 REQUIRED)
+  # Workaround for FindGTest module in CMake older than 3.20
```

Comment:
Wouldn't it be better to use `if(CMAKE_VERSION VERSION_LESS 3.20)` here?

Reply:
I suppose you mean something like
```CMake
if(CMAKE_VERSION VERSION_LESS 3.20)
  set(gtest_target GTest:GTest)
else()
  set(gtest_target GTest::gtest)
endif()
```
but we still use a variable and I find it a bit more subtle (I wasn't even sure that would work, whether the basic signature first search using export mode or module mode) and it would potentially break if the user set [CMAKE_FIND_PACKAGE_PREFER_CONFIG](https://cmake.org/cmake/help/latest/variable/CMAKE_FIND_PACKAGE_PREFER_CONFIG.html#variable:CMAKE_FIND_PACKAGE_PREFER_CONFIG).
I don't feel super strongly about it, I can change it if you want me to.

Reply:
I'm happy with it as is - are we good to merge?

Reply:
@dalg24 another option could be to do:
`
if(CMAKE_VERSION VERSION_LESS 3.20)
  add_library(GTest::gtest ALIAS GTest::GTest)
endif()
```
and then use `GTest::GTest` below.

Reply:
Did you actually try it?  I don't think you can alias an imported target.

Reply:
Ah , ok! Thanks!